gammapy/irf/tests/test_energy_dispersion.py

```diff
@@ -139,3 +139,9 @@ def test_exporter(self):
     @requires_dependency('matplotlib')
     def test_peek(self):
         self.edisp.peek()
+
+    def test_write(self):
+        head = ([('ORIGIN', 'TEST', 'TEST')])
+        hdu = self.edisp.to_table(head)
+        assert hdu.data['ENERGY_LO'][0].all() == self.edisp.energy.lo.value.all()
```

Comment:
This `TestEnergyDispersion2D.test_write` is failing:
https://travis-ci.org/gammapy/gammapy/jobs/302936895#L2666
Please have a look and let me know if it's not clear why.
Maybe it's about lower and upper case mismatch?